src/jobs/generation/MetaGenerator.groovy

```diff
@@ -429,6 +429,8 @@ repos.each { repoInfo ->
                 if (isVSTS) {
                     stringParam('VSTSCollectionName', repoInfo.collection, 'VSTS collection name')
                     stringParam('VSTSCredentialsId', repoInfo.credentials, 'VSTS credentials id')
+                    stringParam('vstsRefspec', '', 'VSTS refspec')
+                    stringParam('vstsBranchOrCommit', '*/master', 'VSTS branch or commit')
```

Comment:
Instead of master, it should be the configured branch for this entry in the MetaGenerator: repoInfo.branch